sale_blanket_order/views/sale_config_settings.xml

```diff
@@ -8,23 +8,18 @@
         <field name="inherit_id" ref="sale.res_config_settings_view_form" />
         <field name="arch" type="xml">
             <xpath
-                expr="//div[hasclass('row','mt16','o_settings_container')][position() = 3]"
+                expr="//block[@name='quotation_order_setting_container']"
```

Comment:
this should suffice: `<block name="quotation_order_setting_container" position="after"></block>`